libraries/ts-command-line/src/CommandLineParameter.ts

```diff
@@ -409,6 +439,14 @@ export class CommandLineIntegerParameter extends CommandLineParameterWithArgumen
   public get value(): number | undefined {
     return this._value;
   }
+
+  /** {@inheritdoc CommandLineParameter.appendToArgList} @override */
+  public appendToArgList(argList: string[]): void {
+    if (this.value !== undefined) {
+      argList.push(this.longName);
+      argList.push(this.value.toString());
```

Comment:
This can probably be in a base class.

Reply:
Not really.  The value types are all different.

---
In reply to: [193924151](https://github.com/Microsoft/web-build-tools/pull/681#discussion_r193924151) [](ancestors = 193924151)